LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
Suggestion: put the MIT license back, unless you want to re-license under something else.

Reply:
MIT is fine. Unless you wanted to go full GPLv3 someday haha.